uint8 imgType
---
sensor_msgs/Image img
